frontend/src/metabase/static-viz/components/XYChart/utils/scales.ts

```diff
@@ -42,7 +42,7 @@ export const createXScale = (
     const xScale = scaleBand({
       domain,
       range,
-      padding: 0.1,
+      padding: 0.2,
```

Comment:
There are a couple of reasons for this change.
1. To make data point values render right in the middle of the waterfall bar. Because in waterfall chart we use `0.2`
   https://github.com/metabase/metabase/blob/a4f301628f152d4a39b5b51feeea7900bd4df05f/frontend/src/metabase/static-viz/components/WaterfallChart/WaterfallChart.jsx#L44
2. To make the bar padding looks similar to the app viz. (Try to open the screenshot in a new tab and compare the bar padding size)
    |                  | Screenshot                                                                                                     |
    |------------------|----------------------------------------------------------------------------------------------------------------|
    | App viz          | ![image](https://user-images.githubusercontent.com/1937582/192002643-d3bfd8f6-9b8a-419a-b8b4-b9e91cbca9c2.png) |
    | static viz (0.1) | ![image](https://user-images.githubusercontent.com/1937582/192002786-46223091-623c-4f41-9397-f77a54beb038.png) |
    | static viz (0.2) | ![image](https://user-images.githubusercontent.com/1937582/192002733-39e7c422-c620-4e4f-b6fd-d23a908a9065.png) |